src/resources/fundingLogos/sofort.jsx

```diff
@@ -0,0 +1,54 @@
+/* @flow */
+/* @jsx jsxToHTML */
+
+import { jsxToHTML, JsxHTMLNode } from '../../lib/jsx'; // eslint-disable-line no-unused-vars
+import { BUTTON_LOGO_COLOR } from '../../constants';
+
+const SOFORT_LOGO_COLORS = {
+    [ BUTTON_LOGO_COLOR.WHITE ]: {
+        primary:   '#ffffff',
```

Comment:
Similar comment as MyBank; these two matching path colors is going to [white-out the logo](https://github.com/paypal/paypal-checkout/blob/d70d9d19ee3e7dfbb9c288e2e43ad939caf2adb0/test/screenshot/images/button.locale%3Dnl_NL%26button.commit%3Dtrue%26button.style.color%3Dblack%26button.style.layout%3Dvertical%26button.style.maxbuttons%3D4%26button.funding.allowed%3Dcredit%2Cideal%2Csofort%2Ccard.png).

Reply:
Also, the Sofort logo looks a bit funky [having a containing box](https://github.com/paypal/paypal-checkout/blob/d70d9d19ee3e7dfbb9c288e2e43ad939caf2adb0/test/screenshot/images/button.locale%3Dnl_NL%26button.commit%3Dtrue%26button.style.color%3Dsilver%26button.style.layout%3Dvertical%26button.style.maxbuttons%3D4%26button.funding.allowed%3Dcredit%2Cideal%2Csofort%2Ccard.png). Is it possible to use a version of this logo without the box?

Reply:
we cannot take out the pink background as that what the legal team wants. as it is their main color